stream-chat-android-client/src/test/java/io/getstream/chat/android/client/api/internal/ExtraDataValidatorTests.kt

```diff
@@ -108,7 +108,7 @@ internal class ExtraDataValidatorTests {
         whenever(chatApi.updateMessage(message)) doReturn message.asCall()
 
         /* When */
-        val result: Result<Message> = validator.updateMessage(message).await()
+        val result: Result<Message> = validator.updateMessage(message,).await()
```

Comment:
Could we remove this comma?

Reply:
I'll remove it, I think it was added either by spotless or automatic refactoring.

Reply:
Updated